docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Kaldi.md

```diff
@@ -50,7 +50,7 @@ The following list provides the Kaldi\*-specific parameters.
 
 ```sh
 Kaldi-specific parameters:
-  --counts COUNTS       A file name with full path to the counts file
+  --counts COUNTS       A file name with full path to the counts file or empty string id counts are included in model file
```

Comment:
I propose to re-write:
A file name with full path to the counts file or empty string to utilize count values from the model file